frontend/packages/console-app/src/components/modals/clone/clone-pvc-modal.tsx

```diff
@@ -125,10 +125,12 @@ const ClonePVCModal = withHandlePromise((props: ClonePVCModalProps) => {
       },
     };
 
-    return handlePromise(k8sCreate(PersistentVolumeClaimModel, pvcCloneObj), (cloneResource) => {
-      close();
-      history.push(resourceObjPath(cloneResource, referenceFor(cloneResource)));
-    });
+    handlePromise(k8sCreate(PersistentVolumeClaimModel, pvcCloneObj))
+      .then((cloneResource) => {
+        close();
+        history.push(resourceObjPath(cloneResource, referenceFor(cloneResource)));
+      })
+      .catch(() => {});
```

Comment:
nit:
I would suggest not to suppress the errors
```suggestion
      .catch((e) => {console.log(e); });
```

Reply:
here and below